### PR TITLE
Improve wrapping of repo header buttons on smaller screens

### LINKED ISSUE #5398: Fix repo title wrapping on small screens
### Description

No words honestly. We reject features and drop content from the header to avoid this stuff and they keep adding buttons. Let's limit the buttons to 50% of the screen and then have them wrap

### Screenshot

Screen Shot 10

### Example URL

https://github.com/refined-github/refined-github

### PR DESCRIPTION
**Pull request by cheap-glitch**
Fixes #5398 

## Test URLs

[refined-github/refined-github](https://github.com/refined-github/refined-github)
[fregante/webext-dynamic-content-scripts](https://github.com/fregante/webext-dynamic-content-scripts)

## Screenshot

**Before** :-1:

before

**After** :+1:

after

### CODE REVIEW

**Review by cheap-glitch — COMMENTED** [review]
Note: as shown in the video above, this PR also restores the buttons on mobile — where they're normally hidden by GitHub.

Not sure if this is a welcomed change, so I don't mind dropping it.

File: source/refined-github.css [hunk 1]
```diff
@@ -177,3 +177,22 @@ pr-branches
...
+		display: flex !important;
+		flex-shrink: 1 !important;
+		flex-wrap: wrap !important;
+		justify-content: flex-end;
+		gap: 8px;
+	}
+
+	.pagehead-actions li {
+		margin-right: 0 !important;
+	}
```
Comment (cheap-glitch) [inline source/refined-github.css:197]: Needed because this is replaced by `gap` on the parent.

**Review by cheap-glitch — COMMENTED** [review]
I think I've reduced the CSS to the minimum amount necessary. Unfortunately it still requires a bit more than 4 lines.

File: source/refined-github.css [hunk 2]
```diff
@@ -177,3 +177,22 @@ pr-branches
 	min-height: 4em;
 	height: 4em;
 }
+
+/* Improve wrapping of repo header buttons on smaller screens #5398 */
+@media (min-width: 768px) {
+	#repository-container-header > :first-child > :first-child {
+		flex-shrink: 0 !important;
+	}
```
[the remark this answers is not in the document]
Comment (fregante) [reply source/refined-github.css:185]: We also have to consider long repository names. `flex-shrink: 3` or higher gives priority to the title without causing the opposite problem.

> not shrink at all

That's the exact reason why this bug exist. This PR causes the opposite problem:

Screen Shot 2

We have to allow wrapping, this is too much code to achieve that

File: source/refined-github.css (hunk 1 above)
Comment (fregante) [reply source/refined-github.css:197]: I'm not sure why but I'm seeing this on Chrome

Screen Shot

Comment (cheap-glitch) [reply source/refined-github.css:197]: That's likely because the `display: flex` didn't apply, the selector just needs to be made more specific.

**Comment by cheap-glitch** [comment]
Changed to the "shrink more strategy" and moved the fix to `github-bugs.css`. Tested on Chromium too:

image

**Comment by cheap-glitch** [comment]
Closing in favor of #5622

**Pull request closed without merging**